CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [19.0.0]
+### Changed
+- **BREAKING**: Split AssetsDetectionController into CollectiblesDetectionController and TokenDetectionController ([#619](https://github.com/MetaMask/controllers/pull/619))
+  - Consumers of the AssetsDetectionController will have to now import both TokenDetectionController and CollectibleDetectionController and split up the calling of any methods accordingly.
+- Update the conversionDate return value to `null` if fetching the data fails.
```

Comment:
We should reference the PR number here as well:
```suggestion
- **BREAKING**: Set the `conversionDate` to `null` if fetching the data fails. (#621 )
```

Also suggested an alternate wording, and marked it as "breaking".